packages/react-sdk/src/components/CallParticipantsList/CallParticipantListingItem.tsx

```diff
@@ -238,90 +264,91 @@ export const ParticipantActionsContextMenu = ({
     <GenericMenu>
       <GenericMenuButtonItem onClick={toggleParticipantPinnedAt}>
         <Icon icon="pin" />
-        {participant.pinnedAt ? 'Unpin' : 'Pin'}
+        {pinnedAt ? t('Unpin') : t('Pin')}
       </GenericMenuButtonItem>
+      <Restricted requiredGrants={[OwnCapability.PIN_FOR_EVERYONE]}>
```

Comment:
Could this 2 buttons be wrapped in a single <Restricted> component?

Reply:
Sure. It is fixed in a recent commit.